org.gitlab.neva_blyad.yat.yml

```diff
@@ -0,0 +1,286 @@
+    build-options:
+      env:
+        - ARCH=x86_64-linux-gnu # TODO: do cross-platform
+    build-commands:
+      - sed -i 's|prefix=/usr|prefix=/app|' wxc.pc.unix
+      - scons
+      - scons install     --prefix=/app/ --arch=$ARCH
+      - scons install-dev --prefix=/app/ --arch=$ARCH
+      - mkdir -p /app/etc/
+      - echo /app/lib/x86_64-linux-gnu/ > /app/etc/ld.so.conf
+    cleanup:
+      - /include
+      - /lib/x86_64-linux-gnu/libwxc.a
+      - /lib/x86_64-linux-gnu/libwxc.so
+      - /lib/x86_64-linux-gnu/pkgconfig
+
+  #-------------------------------------------------------------------------------
+  # libpeas
+  #-------------------------------------------------------------------------------
+  - name: libpeas
+    buildsystem: meson
+    sources:
+      - type: git
+        url: 'https://gitlab.gnome.org/GNOME/libpeas.git'
+        tag: libpeas-1.34.0
+    build-commands:
+      - ln -s /app/share/gir-1.0/Peas-1.0.gir /usr/share/gir-1.0/
+    cleanup:
+      - /bin
+      - /include
+      - /lib/debug
+      - /lib/peas-demo
+      - /lib/pkgconfig
+      - /share
+
+  #-------------------------------------------------------------------------------
+  # meson-perl.
+  #
+  # TODO: patch vs full source.
+  #-------------------------------------------------------------------------------
+  #- name: meson-perl
+  #  buildsystem: meson
+
+  #-------------------------------------------------------------------------------
+  # libpeas-perl.
+  #
+  # TODO
+  #-------------------------------------------------------------------------------
+  #- name: libpeas
+  #  buildsystem: meson
+  #  sources:
+  #    - type: git
+  #      url: 'https://gitlab.com/neva_blyad/libpeas-perl'
+  #      branch: main
+
+  #-------------------------------------------------------------------------------
+  # libgee
+  #-------------------------------------------------------------------------------
+  - name: libgee
+    buildsystem: autotools
+    build-options:
+      env:
+        - ACLOCAL_PATH=/usr/lib/sdk/vala/share/aclocal/
+    sources:
+      - type: git
+        url: 'https://gitlab.gnome.org/GNOME/libgee.git'
+        #branch: master
+        tag: '0.20.6'
+    post-install:
+      - ln -s /app/share/vala/vapi/gee-0.8.vapi /usr/lib/sdk/vala/share/vala-0.56/vapi/
+      - ln -s /app/share/gir-1.0/Gee-0.8.gir /usr/share/gir-1.0/
+    cleanup:
+      - /include
+      - /share
+      - /lib/pkgconfig
+
+  #-------------------------------------------------------------------------------
+  # wxFormBuilder
+  #-------------------------------------------------------------------------------
+  - name: wxformbuilder
+    sources:
+      - type: git
+        url: 'https://github.com/wxFormBuilder/wxFormBuilder'
+        #branch: master
+        tag: v3.9.0
+    buildsystem: simple
+    config-opts:
+      - --disable-mediactrl
+    build-commands:
+      - ./create_build_files4.sh
+      - sed -i "44 a \#include <cstdint>" src/md5/md5.hh
```

Comment:
don't use `sed` to patch. the failure mode is inexistant and it's hard to understand.